apps/arweave/src/ar_data_sync.erl

```diff
@@ -877,6 +890,25 @@ handle_cast({move_data_root_index, Cursor, N}, State) ->
 	move_data_root_index(Cursor, N, State),
 	{noreply, State};
 
+handle_cast({store_data_roots, BlockStart, BlockEnd, TXRoot, Entries}, State) ->
+	#sync_data_state{ store_id = ?DEFAULT_MODULE } = State,
```

Comment:
Should this be part of the `handle_cast` match to avoid a `batmatch`? Or maybe we want a catastrophic `badmatch` because this situation should never occur?

Reply:
Yes, it should never occur.